chemie/web_push/models.py

```diff
@@ -21,18 +21,36 @@ def fifteen_minutes_has_passed(cls):
 
         # Checks if fifteen minutes have passed since last CoffeSubmission object was created
         fifteen_minutes = datetime.timedelta(0, 900)
-        last_coffee_submission_date = (datetime.datetime.utcnow() - fifteen_minutes).replace(tzinfo=pytz.UTC)
+        now = datetime.datetime.now()
+        last_coffee_submission_date = (now - fifteen_minutes).replace(tzinfo=pytz.timezone('Europe/Oslo'))
```

Comment:
In ```settings/base.py```, we have a variable set for this:
```TIME_ZONE = "Europe/Oslo"```

Reply:
I know, men fikk opp feilmelding hvis jeg ikke satt inn timezone. siden datetime.now() ikke setter timzone by default